templates/dhcpd.conf.ntp.erb

```diff
@@ -1,5 +1,5 @@
 # BEGIN NTP Section
-<% if @ntpservers -%>
+<% if @ntpservers.any? -%>
```

Comment:
It's more consistent with other modules to use `<% if not @ntpservers.empty? -%>`